govcd/defined_entity.go

```diff
@@ -615,3 +616,46 @@ func (rde *DefinedEntity) InvokeBehaviorAndMarshal(behaviorId string, invocation
 
 	return nil
 }
+
+// SetAccessControl sets Defined Entity Access Control
+func (de *DefinedEntity) SetAccessControl(acl *types.DefinedEntityAccess) (*types.DefinedEntityAccess, error) {
+	c := crudConfig{
+		endpoint:       types.OpenApiPathVersion1_0_0 + types.OpenApiEndpointRdeEntityAccessControls,
+		endpointParams: []string{de.DefinedEntity.ID},
+		entityLabel:    labelDefinedEntityAccessControl,
+	}
+	return createInnerEntity(de.client, c, acl)
+}
+
+// GetAllAccessControls gets all Defined Entity Access Controls from the receiver DefinedEntity.
+// Query parameters can be supplied to modify pagination.
```

Comment:
"to modify pagination." Shouldn't be "to modify search criteria"? Same for the other occurrences of this comment

Reply:
True, this was bad copy paste.